### PR TITLE
[objc] Add support for more primitive types.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: tests/objc-cli/libmanaged/Tests/Tests.m [hunk 1]
```diff
@@ -52,4 +52,40 @@ - (void)testStaticCallPerformance {
...
+    XCTAssertEqual (UINT_MAX, [Type_UInt32 max], "uint max");
+    
+    XCTAssertEqual (0, [Type_UInt64 min], "ulong min");
+    XCTAssertEqual (ULONG_MAX, [Type_UInt64 max], "ulong max");
+    
+    XCTAssertEqual (-FLT_MAX, [Type_Single min], "single min");
+    XCTAssertEqual (FLT_MAX, [Type_Single max], "single max");
+    
+    XCTAssertEqual (-DBL_MAX, [Type_Double min], "double min");
+    XCTAssertEqual (DBL_MAX, [Type_Double max], "double max");
```
Comment (spouliot) [inline tests/objc-cli/libmanaged/Tests/Tests.m]: missing tests for `NSString`

File: objcgen/objcgenerator.cs [hunk 2]
```diff
@@ -250,14 +250,31 @@ void ReturnValue (Type t)
 		{
 			switch (Type.GetTypeCode (t)) {
 			// unboxing
```
Comment (spouliot) [inline objcgen/objcgenerator.cs]: comment ^ now in wrong place (but can be removed)

File: tests/objc-cli/libmanaged/Tests/Tests.m (hunk 1 above)
Comment (rolfbjarne) [inline tests/objc-cli/libmanaged/Tests/Tests.m]: I forgot, will fix.

Comment (rolfbjarne) [inline tests/objc-cli/libmanaged/Tests/Tests.m]: Fixed.

File: objcgen/objcgenerator.cs (hunk 2 above)
Comment (rolfbjarne) [inline objcgen/objcgenerator.cs]: Removed.

**Pull request merged**